packages/patternfly-4/react-core/src/components/Select/Select.js

```diff
@@ -75,7 +75,7 @@ class Select extends React.Component {
       isExpanded,
       isGrouped,
       selections,
-      ariaLabelledBy,
+      'aria-labelledby': ariaLabelledBy,
```

Comment:
Same issue as previous comment... The old prop ariaLabelledBy is still being used in the example.